app/models/cohort.rb

```diff
@@ -3,13 +3,13 @@
 # Table name: cohorts
 #
 #  id          :bigint           not null, primary key
-#  program_id  :integer
+#  program     :bigint           not null
```

Comment:
You removed the _id at the end of program as well as creator and contact. Was that intentional?